lib/tasks/deliver.rake

```diff
@@ -1,15 +1,17 @@
 namespace :deliver do
-  def test_email(address)
+  def test_email(address, subscriber_id)
```

Comment:
This is the bit that causes the problem.

Reply:
Oh oops, I'll raise a PR to make it optional.

Reply:
https://github.com/alphagov/email-alert-api/pull/549